.github/workflows/acceptance-tests-runner.yml

```diff
@@ -382,6 +382,38 @@ jobs:
             ./internal/service/advancedclustertpf
         run: make testacc
 
+  advanced_cluster_tpf_move:
+    needs: [ change-detection, get-provider-version ]
+    if: ${{ (needs.change-detection.outputs.advanced_cluster_tpf == 'true' || inputs.test_group == 'advanced_cluster_tpf') && inputs.terraform_version == '1.10.x' }}
+    env:
+      MONGODB_ATLAS_ADVANCED_CLUSTER_V2_SCHEMA: 'true'
+    strategy:
+      max-parallel: 2
```

Comment:
why 2? Could it be 10?

Reply:
yes, i thought exactly about saying 10, but right we only have 2 elements so i thought it could be confusing.  effectively it's running everything in parallel